Makefile

```diff
@@ -116,11 +116,23 @@ generate: $(CONTROLLER_GEN) ## Generate code
 
 .PHONY: generate-deepcopy
 generate-deepcopy: $(CONTROLLER_GEN) ## Generate deepcopy files
-	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate/boilerplate.generatego.txt paths=./api/...
+	$(CONTROLLER_GEN) \
+		object:headerFile=./hack/boilerplate/boilerplate.generatego.txt \
+		paths=./api/... \
+		crd:preserveUnknownFields=false \
+		output:crd:dir=./config/crd/bases
```

Comment:
I believe this should be missing lines 122 and 123 here, since it looks like the intent is to only generate the deepcopy code here.

Reply:
removed them